app/javascript/packs/reducers/restaurants/actions.js

```diff
@@ -0,0 +1,47 @@
+import { getFiltersHash } from './selectors';
+import { searchRestaurants as searchRestaurantsApi } from '../../api/restaurants';
+import { hash } from '../../modules/hash';
+
+const ACTIONS_NAME = {
+    SEARCH_RESTAURANTS_STARTED: 'restaurants-search-started',
+    SEARCH_RESTAURANTS_DONE: 'restaurants-search-done',
+    SEARCH_RESTAURANTS_FAILED: 'restaurants-search-failed',
+};
```

Comment:
Have you ever used [`keyMirror`](https://github.com/STRML/keyMirror) or any of those?
They're a bit controversial nowadays, but still useful, I think.

Reply:
Not a fan, to me it looks cleaner to do `a = { a: 'a', ASD: 'ASD'}`